client/js/options.js

```diff
@@ -26,6 +27,7 @@ const options = $.extend({
 	thumbnails: true,
 	userStyles: userStyles.text(),
 	highlights: [],
+	highlightsRE: [],
```

Comment:
This should be an empty regex or something because `options.highlightsRE.exec` may fail.